storycraftr/llamaindex/chat_integration.py

```
@@ -0,0 +1,357 @@
+    
+    # Parse command and args
+    parts = command.split()
+    if not parts:
+        return show_llamaindex_help()
+    
+    cmd = parts[0].lower()
+    args = parts[1:]
+    
+    # Handle commands
+    if cmd == "build" or cmd == "build-index":
+        return handle_build_command(args, book_path)
```

Ruff (0.8.2)

Undefined name `handle_build_command`

(F821)